engine/flink/executor/src/main/scala/pl/touk/nussknacker/engine/process/compiler/TestFlinkProcessCompiler.scala

```diff
@@ -64,7 +64,10 @@ class TestFlinkProcessCompiler(creator: ProcessConfigCreator,
 
   private def prepareDataForTest[T](sourceTestSupport: SourceTestSupport[T], scenarioTestData: ScenarioTestData, sourceId: NodeId): List[T] = {
     val testParserForSource = sourceTestSupport.testRecordParser
-    val testRecordsForSource = scenarioTestData.testRecords.filter(_.sourceId == sourceId).map(_.record)
+    val testRecordsForSource = scenarioTestData.testRecords
+      .collect{ case testRecord: ScenarioTestJsonRecord => testRecord}
```

Comment:
I think we should add support also for Flink based engine?

Reply:
I've added support for flink based engine :+1: